README.md

```diff
@@ -19,55 +17,75 @@ Build a RESTful API that can `get/create/update/delete` user data from a persist
 }
 ```
 
-## Requirements
-
-### Functionality
+## Installation
 
-- The API should follow typical RESTful API design pattern.
-- The data should be saved in the DB.
-- Provide proper unit test.
-- Provide proper API document.
+First install Node.js and MongoDB. Then:
 
-### Tech stack
+```bash
+npm install
+```
 
-- Use Node.js and any framework.
-- Use any DB. NoSQL DB is preferred.
+## Getting Started
 
-### Bonus
+```bash
+npm start
+```
 
-- Write clear documentation on how it's designed and how to run the code.
-- Write good in-code comments.
-- Write good commit messages.
-- An online demo is always welcome.
+## API
 
-### Advanced requirements
+### Some HTTP Status
 
-*These are used for some further challenges. You can safely skip them if you are not asked to do any, but feel free to try out.*
+```
+200 ok
+201 ok, created
+204 ok, no content
+400 bad request, like validate error
+401 unauthorized
+404 not found
+500 server error
+```
 
-- Provide a complete user auth (authentication/authorization/etc.) strategy, such as OAuth.
-- Provide a complete logging (when/how/etc.) strategy.
-- Imagine we have a new requirement right now that the user instances need to link to each other, i.e., a list of "followers/following" or "friends". Can you find out how you would design the model structure and what API you would build for querying or modifying it?
-- Related to the requirement above, suppose the address of user now includes a geographic coordinate(i.e., latitude and longitude), can you build an API that,
-  - given a user name
-  - return the nearby friends
+### Details
 
+#### Login
 
-## What We Care About
+```
+POST /login HTTP/1.1
+Content-Type: application/json
+{
+    "user": "admin",
+    "password": "123456"
+}
 
-Feel free to use any open-source library as you see fit, but remember that we are evaluating your coding skills and problem solving skills.
+Response Body Like:
+{
+    "access_token": "xxx...",
+    "token_type": "bearer",
+    "expires_in": 7200
```

Comment:
how to configure the `expires_in` time

Reply:
and how to configure the jwt key

Reply:
Could config in config.js, maybe get from environment variables in production.